apis/v1beta1/validation/httproute_test.go

```diff
@@ -445,6 +445,58 @@ func TestValidateHTTPRoute(t *testing.T) {
 				},
 			}},
 		}},
+	}, {
+		name:     "multiple actions for the same request header (invalid)",
+		errCount: 2,
+		rules: []gatewayv1b1.HTTPRouteRule{{
+			Filters: []gatewayv1b1.HTTPRouteFilter{{
+				Type: gatewayv1b1.HTTPRouteFilterRequestHeaderModifier,
+				RequestHeaderModifier: &gatewayv1b1.HTTPHeaderFilter{
+					Add: []gatewayv1b1.HTTPHeader{
+						{
+							Name:  gatewayv1b1.HTTPHeaderName("x-fruit"),
```

Comment:
I think we're still missing the following test cases:

1. Repeat names within the same list (Add, Set, Remove)
2. Case insensitive matching ("x-fruit" and "X-fruit" should still be invalid)

Reply:
You would think so for the first one, but it's [apparently a schema violation](https://github.com/kubernetes-sigs/gateway-api/pull/1497#issuecomment-1297827635). I can run it through `Validate()`, but it won't ever occur in practice. Should the tests include configuration they should never receive?

Reply:
Repeats of the same are now in, but note that you shouldn't see these in practice as they [violate the schema](https://github.com/kubernetes-sigs/gateway-api/pull/1497#issuecomment-1297827635)